### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by qbsale** [florisboard/florisboard#2691] [Q&A] [answered]
Hi Patrick, I really love your work, I would like to know if you could send me the documentation in order to create my own custom keyboard. I can't really find anything useful on web. I cannnot figure it out how to associate the onkeypress() and the swipe to special character and symbol layout.

**Comment by patrickgold** [comment, marked as answer]
Hi,
there isn't a single place to go for documentation, this keyboard is a result
of the official Android documentation, existing knowledge and looking at LatinIME/other open source keyboards.
Hope that helps!
